concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -307,7 +307,7 @@ Hale-1973-1798-305	Animals-8	pheasant	n	03a.008	3060	PHEASANT
 Hale-1973-1798-306	Animals-9	peacock	n	03a.009	3382	PEACOCK
 Hale-1973-1798-307	Animals-10	sparrow	n	03a.010	1854	SPARROW
 Hale-1973-1798-308	Animals-11	swallow	n	03a.011	3077	SWALLOW (BIRD)
-Hale-1973-1798-309	Animals-12	bill	n	03a.012		
+Hale-1973-1798-309	Animals-12	bill	n	03a.012	1081	BILL
```

Comment:
Since this is in the category "Animals", eked in between bird related terminology, I assume this must mean "the jaws of a bird together with their horny covering " (https://www.merriam-webster.com/dictionary/bill). The definition of `1081 BILL` however reads "Sum owed in a restaurant, or the relative bill." (https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1082). Would unmap.